### DISCUSSION TITLE
[Kanban] Card/column title as link to corresponding tiddler

### DISCUSSION THREAD

**Discussion by AshRubigo** [BurningTreeC/tiddlywiki-muuri#15] [Ideas]
Currently the search feature must be used to access a card or column tiddler outside the kanban board interface.

It would be more convenient if the title of cards and columns behaved as links to the corresponding tiddler that open normally per the current story view.

**Comment by BurningTreeC** [comment]
That's a good idea @AshRubigo 

I'll add that feature to the Kanban plugin

Thanks!

**Reply by AshRubigo** [reply]
I've just noticed the permalink button of cards effectively has this functionality, though it is not present for lists.
